relay/relay.py

```diff
@@ -110,6 +132,38 @@ def get_networks_of_user(self, user_address: str) -> List[str]:
                 networks_of_user.append(network_address)
         return networks_of_user
 
+    def get_network_event_queries(self, user_address: str, type: str, from_block: int):
```

Comment:
if you put that function here it should also execute the queries and not only give the queries back.